go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/external-secrets/external-secrets
 
 go 1.23.1
 
+toolchain go1.23.3
```

Comment:
Please remove this.

Reply:
sorry, I removed it
I had some issues locally and did some clean up and this probably appeared because of it.